monai/apps/auto3dseg/auto_runner.py

```diff
@@ -767,20 +771,34 @@ def run(self):
                     "Possibly the required algorithms generation step was not completed."
                 )
 
-            if not self.hpo:
-                self._train_algo_in_sequence(history)
-            else:
-                self._train_algo_in_nni(history)
+            if auto_train_choice:
+                history = [h for h in history if not h["is_trained"]]  # skip trained
```

Comment:
Shall we have a logging at the debug level for the algorithms picked for training, or the algorithms skipped because `is_trained` is true?

Reply:
added logging